Python/libraries/recognizers-date-time/recognizers_date_time/date_time/base_time_zone.py

```diff
@@ -0,0 +1,36 @@
+from typing import List, Pattern
```

Comment:
Check for unused imports, i dont think u are using `Pattern`

Reply:
My bad